tests/accelerators/test_accelerator_connector.py

```diff
@@ -437,13 +437,16 @@ def test_ipython_incompatible_backend_error(*_):
     with pytest.raises(MisconfigurationException, match="backend ddp is not compatible"):
         Trainer(accelerator="ddp", gpus=2)
 
-    with pytest.raises(MisconfigurationException, match="backend ddp is not compatible"):
-        Trainer(accelerator="ddp_cpu", num_processes=2)
-
     with pytest.raises(MisconfigurationException, match="backend ddp2 is not compatible"):
         Trainer(accelerator="ddp2", gpus=2)
 
 
+@mock.patch("pytorch_lightning.utilities._IS_INTERACTIVE", return_value=True)
+def test_ipython_compatible_bacend(*_):
+    # Runs withoutfailing
```

Comment:
```suggestion
    # Runs without failing
```
or better to remove the comment because it's odd to state that the test should pass. Every test in the test suite is expected to pass without failing :)))